batch/sql/estimated-current.sql

```diff
@@ -257,7 +257,7 @@ CREATE TABLE IF NOT EXISTS `jobs` (
 CREATE INDEX `jobs_batch_id_state_always_run_inst_coll_cancelled` ON `jobs` (`batch_id`, `state`, `always_run`, `inst_coll`, `cancelled`);
 CREATE INDEX `jobs_batch_id_state_always_run_cancelled` ON `jobs` (`batch_id`, `state`, `always_run`, `cancelled`);
 CREATE INDEX `jobs_batch_id_update_id` ON `jobs` (`batch_id`, `update_id`);
-CREATE INDEX `jobs_batch_id_always_run_n_regions_regions_bits_rep_job_id` ON `jobs` (`batch_id`, `always_run`, `n_regions`, `regions_bits_rep`, `job_id`);
```

Comment:
Should this change drop this index?

Reply:
In thinking it through further, I think I'd like to do that in a separate PR. It seems dangerous to delete a critical index until the service is using the new one in case there's problems. What do you think?

Reply:
Makes sense! Can do it that way